e2e/interchaintestv8/ibc_eureka_test.go

```diff
@@ -72,7 +72,7 @@ type IbcEurekaTestSuite struct {
 
 // SetupSuite calls the underlying IbcEurekaTestSuite's SetupSuite method
 // and deploys the IbcEureka contract
-func (s *IbcEurekaTestSuite) SetupSuite(ctx context.Context) {
+func (s *IbcEurekaTestSuite) SetupSuite(ctx context.Context, pt operator.SupportedProofType) {
```

Comment:
Since this is a sent-in parameter that is used a long way from where we see the type, I would prefer we just spell out the name to proofType. Makes the code further down more readable.